### ISSUE TITLE
gqlparser/v2 2.2.0 Incompatability

### ISSUE THREAD

**Issue by devotoare** [99designs/gqlgen#1516]
### What happened?
Updated `github.com/vektah/gqlparser/v2 v2.1.0` to `github.com/vektah/gqlparser/v2 v2.2.0` and the generate command started spitting out garbage:

```
go run github.com/99designs/gqlgen generate 
validation failed: packages.Load: /project/graph/prelude.resolvers.go:19:44: __DirectiveResolver not declared by package generated
exit status 1
```

Downgrading to v2.1.0 and removing the new `prelude.resolvers.go` file worked fine.

### What did you expect?
Generate command to work.

### Minimal graphql.schema and models to reproduce
Any

### versions
 - `gqlgen version`: 0.13.0
 - `go version` : 1.16.3
 - dep or go modules: go modules

**Comment by lwc** [comment]
Hello, if you update gqlgen to the latest master and remove prelude.resolvers.go does the problem persist?

**Comment by devotoare** [comment]
@lwc Just tested that, no the problem does not persist, works as expected.

**Comment by YannHulot** [comment]
I had the same issue. I reverted to 2.1.0 and it works. I 'll hold off updating this dep for a while until things settle down.

**Comment by odmishien** [comment]
I had the same issue. and in my case, Downgrading to v2.1.0 and removing the new prelude.resolvers.go file didn't work fine....:(

```
gqlgen generate                                            
validation failed: packages.Load: /Users/mishima/.ghq/github.com/odmishien/graphql-station-tutorial/graph/prelude.resolvers.go:19:44: __DirectiveResolver not exported by package generated
```

**Comment by dailytravel** [comment]
go get github.com/vektah/gqlparser/v2@v2.1.0 and remove file prelude.resolvers.go, it's well!

**Comment by razorness** [comment]
Build fails with current master:

```
..\graph\resolver\prelude.resolvers.go:19:34: cannot refer to unexported name generated.__DirectiveResolver
```

**Comment by syssam** [comment]
@vektah would you follow up?

**Comment by pablote** [comment]
is there a fix for this besides reverting to 2.1.0?

**Comment by benjaminjkraft** [comment]
This looks to be fixed in master by f57d1a028.

**Comment by Araly** [comment]
I still get the same issue as of now, when using gqlparser/v2 v2.2.0, specifically using https://github.com/Yamashou/gqlgenc if it's any relevant

**Comment by dailytravel** [comment]
Hi!

You need downgrade v2.1.0, good luck!

**Comment by lwc** [comment]
Fixed in v0.14.0
